de.haeckerfelix.Shortwave.json

```diff
@@ -1,50 +1,79 @@
 {
-    "app-id" : "de.haeckerfelix.Shortwave",
-    "runtime" : "org.gnome.Platform",
-    "runtime-version" : "45",
-    "sdk" : "org.gnome.Sdk",
-    "sdk-extensions" : ["org.freedesktop.Sdk.Extension.rust-stable"],
-    "command" : "shortwave",
-    "finish-args" : [
+    "id": "de.haeckerfelix.Shortwave",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "47",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-stable"
```

Comment:
I think you need to add LLVM here as well as to `append-path`

https://github.com/flathub/org.gnome.gitlab.YaLTeR.Identity/blob/master/org.gnome.gitlab.YaLTeR.Identity.json#L8